Gopkg.toml

```diff
@@ -26,7 +26,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "SAAS-93-Configurable-Action-Timeout"
```

Comment:
change after https://github.com/percona/pmm/pull/565 gets merged

Reply:
It is merged